src/main/resources/application.yaml

```diff
@@ -89,7 +89,7 @@ security:
 trusted:
   s2s:
     service:
-      names: ccpay_bubble,cmc,bulk_scan_payment_processor,api_gw,probate_frontend,divorce_frontend,ccd_gw,payment_app
+      names: ccpay_bubble,cmc,bulk_scan_payment_processor,api_gw,probate_frontend,divorce_frontend,ccd_gw,payment_app,api_gw
```

Comment:
Hi @YasharRahvar - If possible please introduce a variable for services list in values.yaml and refer it here instead of hardcoding it. So that we can handle the values in cnp-flux.